### DISCUSSION TITLE
[typescript] intersection type으로 리턴하기

### DISCUSSION THREAD

**Discussion by gangukss** [42-Tomodachi/ft_transcendence#9] [General]
리턴 타입에 인터섹션 타입으로 User, accessToken 묶어서 주고싶은데 문법 어떻게 해야하는지 아시는분 있으신가요~
![image](https://user-images.githubusercontent.com/59438936/173191352-342c88d7-d5a2-4473-9a46-3b7653a53b49.png)

**Comment by gangukss** [comment]
User 가 엔티티여서 BaseEntity 를 상속받아 구현되었습니다. 그래서 관련된 메서드가 정의되지 않아 뜨는 오류였던 것 같습니다. Partial<User> 처럼 사용하면 에러 없이 사용 가능할 것 같습니다.

**Reply by som-syom** [reply]
올려달라고 하고 잊고있다가 생각나서 왔는데 해결되었네요 ㅎㅎ;; 👍
